features/install.feature

```diff
@@ -5,7 +5,45 @@ Feature: Installation
   So that I can start testing without repetitive setup work.
 
 
-  Scenario: fresh install
+  Scenario: install with a .js test helper
+    Given I am working on a Node.js project
+    When I run "mycha install" and enter "spec/spec_helper.js"
+    Then my "package.json" now lists the devDependencies
+      | NPM MODULE |
+      | chai       |
+      | sinon      |
+      | sinon-chai |
+    And my project now has a file "spec/spec_helper.js" containing
+      """
+      var chai = require('chai');
+      var sinon = require('sinon');
+      chai.use(require('sinon-chai'));
+
+      global.chai = chai;
+      global.expect = chai.expect;
+      global.sinon = sinon;
+
+      process.env.NODE_ENV = 'test';
```

Comment:
I think its best to move this to set this as early as possible, i.e. move this to the first line in the file.